mongo-helm/templates/mongo-secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: mongodb-secret
+type: Opaque
+data:
+    mongo-root-username: dXNlcm5hbWU=
+    mongo-root-password: cGFzc3dvcmQ=

```

Comment:
Please avoid pushing Secret manifests to version control. They contain sensitive info.

Reply:
Great point, fixed.